src/ralph/assets/utils.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+
+
+def move_parents_models(from_obj, to_obj):
+    """
+    Move parents Django model to diffrent model inheriting from
+    the same parent.
+
+    By default Django does not allow the remove of the model without parents.
+    Clean _meta.parents, we remove object and add back default parents to class
+
+    Args:
+        from_obj: From Django model object
+        to_obj: To Django modelo object
+
+    Example:
+        >> instance = DataCenterAsset.objects.get(pk=1)
+        >> back_office_asset = BackOfficeAsset()
+        >> move_parents_models(instance, back_office_asset)
+    """
+    to_obj.__dict__.update(from_obj.__dict__)
+    to_obj.save()
+    parents = from_obj._meta.parents
+    from_obj._meta.parents = {}
+    from_obj.delete()
```

Comment:
what if there will be some exception during deletion? parent will be empty for all further objects. this should be wrapped in try-finally imo